gulpfile.babel.js

```diff
@@ -1,334 +1,45 @@
 /**
  * External dependencies
  */
-import autoprefixer from 'gulp-autoprefixer';
 import babel from 'gulp-babel';
-import banner from 'gulp-banner';
 import check from 'gulp-check';
-import cleanCSS from 'gulp-clean-css';
 import colors from 'ansi-colors';
+import fs from 'fs';
```

Comment:
Nitpick -- out of alphabetical order :grin:

Reply:
Oh, so is `eslint` a bit further down. Nevermind I guess.

Reply:
In for a penny - in for a pound :)